<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
    border-radius: 10px;             /*四角圆角的直径*/
    border:1px solid rgb(27,27,27);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background-color: rgb(0, 0, 0);
    border-bottom-left-radius: 7px;
    border-bottom-right-radius: 7px;

border-bottom:1px solid rgb(100,100,100);
border-left:1px solid rgb(100,100,100);
border-right:1px solid rgb(100,100,100);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="tePictureListView" name="picturelist">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border:1px solid #4c399e;
background-color: rgb(27, 27, 27);
}
QFrame:hover{
    border-color: #6045d6;
}
QFrame:!hover{
    border-color: #4c399e;
}
QFrame[saving=&quot;true&quot;]{
    border-color: #0af581;
}
QScrollBar{background:#5421ff;}
QScrollBar::vertical{margin: 12px 0px 12px 0px;border:0px solid #5421ff;width:10px;}
QScrollBar::horizontal{margin: 0px 12px 0px 12px;border:0px solid #5421ff;height:10px;}
QScrollBar::handle:!pressed{background:#7751f3;border:1px solid #c5a9f7;}
QScrollBar::handle:pressed{background:#c5a9f7;}

QScrollBar::handle:vertical{min-height:20px;}
QScrollBar::handle:vertical{min-width:20px;}
QScrollBar::up-arrow:vertical{
    border-image: url(&quot;:/res/scrollbar_uparrow.png&quot;);
    width:10px;height:10px;
}
QScrollBar::down-arrow:vertical{
    border-image: url(&quot;:/res/scrollbar_downarrow.png&quot;);
    width:10px;height:10px;
}

QScrollBar::left-arrow:horizontal {
    border-image: url(&quot;:/res/scrollbar_leftarrow.png&quot;);
    width:10px;height:10px;
}
QScrollBar::right-arrow:horizontal {
    border-image: url(&quot;:/res/scrollbar_rightarrow.png&quot;);
    width:10px;height:10px;
}
QScrollBar::sub-page,QScrollBar::add-page {
    background-color: #6d6292;
}
QScrollBar::sub-line:vertical {
    subcontrol-position: top;
    subcontrol-origin: margin;
    background-color: #6644d6;
    border: 0px solid #e7dfff;
    height:12px;
    width:10px;
}
QScrollBar::add-line:vertical {
    subcontrol-position: bottom;
    subcontrol-origin: margin;
    background-color: #6644d6;
    border: 0px solid #e7dfff;
    height: 12px;
    width:10px;
}

QScrollBar::sub-line:horizontal {
    subcontrol-position: left;
    subcontrol-origin: margin;
    background-color: #6644d6;
    border: 0px solid #e7dfff;
    height:10px;
    width:12px;
}
QScrollBar::add-line:horizontal {
    subcontrol-position: right;
    subcontrol-origin: margin;
    background-color: #6644d6;
    border: 0px solid #e7dfff;
    height: 10px;
    width:12px;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="TaglistTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane:hover { /* The tab widget frame */
    border-top: 1px solid #6045d6;
}
QTabWidget::pane:!hover {
    border-top: 1px solid #4c399e;
}
QTabWidget::tab-bar {
    left: 0px;
}
/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    color:white;
    background:#5e4e7b;
    border: 1px solid #c5aaf7;
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    min-width: 8ex;
    padding: 2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
    background:#4d7d74;
    border-color: #8ce5d5;
}
QTabBar::tab:selected {
    margin-left: -4px;
    margin-right: -4px;
    border-color: #73f3b3;
    background: #3a604d;
}
QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}
QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="selectList_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(27, 27, 27);</string>
       </property>
       <attribute name="title">
        <string>Select</string>
       </attribute>
       <layout class="QVBoxLayout" name="selectList_tab_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="teTagListWidget" name="taglist" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="allList_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(27, 27, 27);</string>
       </property>
       <attribute name="title">
        <string>All</string>
       </attribute>
       <layout class="QVBoxLayout" name="allList_tab_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="teMultitagListView" name="GlobalMultiTaglistView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="teEditorList" name="editorlist" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{
border-top-left-radius: 7px;
border-top-right-radius: 7px;
color:  #d2c5ff ;
background-color: rgb(27, 27, 27);
border-top:1px solid rgb(100,100,100);
border-left:1px solid rgb(100,100,100);
border-right:1px solid rgb(100,100,100);
}
QMenuBar::item:selected{

	background: #613d80;
}
QMenuBar::item:first:selected{
    border-top-left-radius: 7px; 
	background: #613d80;
}
/*QMenuBar::item:first:selected{
	background: #613d80;
	border-top-left-radius: 7px; 
}*/
QMenu{
color:  #d2c5ff ;
background-color: rgb(27, 27, 27);
selection-background-color: rgb(70, 70, 70);
}</string>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_add"/>
    <addaction name="action_close"/>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="action_editor_setting"/>
    <addaction name="actionAuto_Merge_Tags"/>
    <addaction name="actionAuto_Save_State"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_filt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_editor_setting">
   <property name="text">
    <string>Editors</string>
   </property>
  </action>
  <action name="action_filt">
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionAuto_Merge_Tags">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Merge Tags</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Clear EditorList(Debug)</string>
   </property>
  </action>
  <action name="action_Remove_a_TagList_Layout_Item">
   <property name="text">
    <string>Remove a TagList Layout Item</string>
   </property>
  </action>
  <action name="actionAuto_Save_State">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Save State</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>teTagListWidget</class>
   <extends>QWidget</extends>
   <header>tetaglistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>teEditorList</class>
   <extends>QWidget</extends>
   <header>teeditorlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>teMultitagListView</class>
   <extends>QListView</extends>
   <header>temultitaglistview.h</header>
  </customwidget>
  <customwidget>
   <class>tePictureListView</class>
   <extends>QListView</extends>
   <header>tepicturelistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
